packages/php/src/Webhooks.php

```diff
@@ -0,0 +1,46 @@
+<?php
+
+namespace ReadMe;
+
+class Webhooks
```

Comment:
Figured it was cleaner to have this be a separate class instead of calling `\ReadMe\Metrics::verifyWebhook()` as this doesn't do any metricsy things.

Reply:
Yeah think this is what i did for .net too: https://github.com/readmeio/metrics-sdks/blob/55fa6ac31f84a7ce93aceeb2bcd310ebdd6dfd9a/packages/dotnet/examples/net6.0-webhook/Program.cs#L24